helpers/mail/mail_v3_test.go

```diff
@@ -438,9 +438,22 @@ func TestV3PersonalizationSetSendAt(t *testing.T) {
 	}
 }
 
+func TestV3NewAttachmentFromFile(t *testing.T) {
+	file := "attachment_test.txt"
+	a := NewAttachmentFromFile(file)
+	if a.Content != "SGVsbG8gd29ybGQhCg==" {
+		t.Errorf("a.Content was `%s`. Got %s.", a.Content, "SGVsbG8gd29ybGQhCg==")
+	}
+
+	file = "attachment_test_utf8.txt"
```

Comment:
could you split this test into 2 tests? You could call the second test something like `TestV3NewAttachmentFromFileUTF8` or `TestV3NewAttachmentFromFileUnicode`.